### DISCUSSION TITLE
Web Shop Function - Additional Options for Restrictions

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#577] [Suggestions]
Currently, we may be restricted to one cluster, or server, versus having the option to select multiples in a way that reflects this on the *web* shop. If I needed an option to restrict a shop pack to multiple clusters and have it reflect in the web shop's filtering, I don't have that option. What I am requesting is the option the 'checkbox' multiple servers/clusters that the particular shop pack is restricted *to*, so that these packs can be queried with filters for those servers/clusters on the website; shop packs that are available for an ASE PVE and PVP server, but not an ASA PVE server, etc. 
![image](https://github.com/gameserverapp/Platform/assets/44717221/b433bdaa-460e-42de-b860-3960780697ad)
![image](https://github.com/gameserverapp/Platform/assets/44717221/6065dee1-2ed6-4591-99cc-356e81ba1aba)

**Comment by mvaessen** [comment]
Have you considered using the other option available, that lets you select multiple servers 
![Screenshot 2024-09-27 at 11 37 30](https://github.com/user-attachments/assets/18c53f46-d114-4ed6-a596-062669ffc442)

**Reply by Ian-Highlander** [reply]
> Have you considered using the other option available, that lets you select multiple servers ![Screenshot 2024-09-27 at 11 37 30](https://private-user-images.githubusercontent.com/13402423/371478462-18c53f46-d114-4ed6-a596-062669ffc442.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc1MTMzODgsIm5iZiI6MTcyNzUxMzA4OCwicGF0aCI6Ii8xMzQwMjQyMy8zNzE0Nzg0NjItMThjNTNmNDYtZDExNC00ZWQ2LWE1OTYtMDYyNjY5ZmZjNDQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI4VDA4NDQ0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlkOTRlMGJkZGJmZmU4NTE1ZGRiZmVkYzlkOWZmYmRmYzY3ZGE4NDAyYzJlZmY2YjMwYzEwM2YzY2YzNzA2NzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.45H3lSm40OVpZ4dKpoHmtFxUEmEgtWgYpWvh_zs0Z1Y)

This would result in potentially having to edit every single relevant shop item when a new server is added (regularly for Ark with Wildcard releasing new maps every couple of months). It also doesn't cover the scenario I gave in https://github.com/gameserverapp/Platform/discussions/811#discussion-6439289 which you've dismissed as a duplicate of this one (it's similar for sure but not exactly the same).
